assets/js/modules/analytics-4/components/dashboard/ActivationBanner/index.stories.js

```diff
@@ -16,15 +16,37 @@
  * limitations under the License.
  */
 
+/**
+ * External dependencies
+ */
+import { waitForElement } from '@testing-library/react';
+
 /**
  * Internal dependencies
  */
 import ActivationBanner from './index';
 
 const Template = () => <ActivationBanner />;
 
-export const Default = Template.bind( {} );
-Default.storyName = 'Default';
+export const ReminderBanner = Template.bind( {} );
+ReminderBanner.storyName = 'ReminderBanner';
+
+export const SetupBanner = Template.bind( {} );
+SetupBanner.storyName = 'SetupBanner';
+SetupBanner.decorators = [
```

Comment:
Let's create stories for each case that we can have based on measurement ID and existing tag combinations.